frontend/server/src/Controllers/Contest.php

```diff
@@ -2671,6 +2675,18 @@ public static function apiCreate(\OmegaUp\Request $r) {
             ['partial','all_or_nothing','max_per_group'],
         );
         $checkPlagiarism = $r->ensureOptionalBool('check_plagiarism') ?? false;
+        
+        // Handle recommended flag - only available for admins and support team
+        if (
+            !is_null($r->ensureOptionalBool('recommended')) &&
+            !\OmegaUp\Authorization::isSystemAdmin($r->identity) &&
+            !\OmegaUp\Authorization::isSupportTeamMember($r->identity)
```

Comment:
Sysadmins should have support team member role, so line 2682 can be deleted:
```suggestion
            !is_null($r->ensureOptionalBool('recommended')) &&
            !\OmegaUp\Authorization::isSupportTeamMember($r->identity)
```